pkg/controller/configmap/configmap.go

```diff
@@ -66,8 +66,14 @@ func (r *ReconcileConfigMap) syncConfigMap(
 				reqLogger,
 				configmap,
```

Comment:
This arg should be kubecluster.

Reply:
why will syncConfigMap take kubecluster??

Reply:
The second argument to LogInfof is the object to post the event to.